scripts/seed.ts

```diff
@@ -66,6 +66,13 @@ export default async () => {
 
     // Note: if using PostgreSQL, using `createMany` to insert multiple records is much faster
     // @see: https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#createmany
+    await Promise.all(
+      users.map(async (data: Prisma.UserCreateArgs['data']) => {
+        const record = await db.user.create({ data })
+        console.log(record)
+      })
+    )
+
```

Comment:
Per the previous comment, this should be moved closer to where the users are being defined instead of being separated by the `inputTemplates` and `outputTemplates` definitions.
```suggestion
```